### ISSUE TITLE
Hyperparameter search

### ISSUE THREAD

**Issue by joannacknight** [alan-turing-institute/ARC-MTQE#49]
Continue the hyperparameter search started in issue #23 now that the code for the loss functions has been updated.
